server/app.py

```diff
@@ -157,12 +165,78 @@ def get_schools():
     response.headers['Access-Control-Allow-Origin'] = '*'
     return response
 
+@app.route('/position', methods=['POST'])
+def create_position():
+    if request.mimetype != 'application/json':
+        raise Exception('Content-Type is not "application/json".')
+    j = request.get_json()
+    Session, engine = dbconnect(db_options)
+    session = Session()
+    position_id = get_or_create_position(session, j['position_type'])
+    session.commit()
+    return jsonify(
+        {
+            'id': position_id
+        }
+    )
+
+@app.route('/position', methods=['GET'])
+def get_positions():
+    Session, engine = dbconnect(db_options)
+    session = Session()
+    positions = session.query(Position).all()
+    print (positions)
+    formatted_positions = []
+    for p in positions:
+        formatted_positions.append({
+            'position_type': p.position_type
+        })
+    response = Response(json.dumps(formatted_positions))
+    response.headers['Access-Control-Allow-Origin'] = '*'
+    return response
+
+@app.route('/adminuser', methods=['POST'])
+def create_admin():
+    if request.mimetype != 'application/json':
+        raise Exception('Content-Type is not "application/json".')
+    j = request.get_json()
+    Session, engine = dbconnect(db_options)
+    session = Session()
+    admin = UserClubPositionMapping(
+        user_id=j.get('user_id'),
+        club_id=j.get('club_id'),
+        position_id =j.get('position_id')
+    )
+    session.add(admin)
+    session.flush()
+    session.commit()
+    return jsonify(
+        {
+            'id': admin.id
+        }
+    )
+
+@app.route('/adminuser', methods=['GET'])
+def get_admins():
+    Session, engine = dbconnect(db_options)
+    session = Session()
+    admins = session.query(UserClubPositionMapping).all()
+    print (admins)
+    formatted_admins = []
+    for a in admins:
+        formatted_admins.append({
+            'user_id': a.user_id,
```

Comment:
probably want to return the actual user information, so you're going to have to query the user table